client/src/components/manageProjects/createNewEvent.js

```diff
@@ -103,6 +110,10 @@ const CreateNewEvent = ({
             type="button"
             className="create-form-button"
             onClick={() => {
+              if (formValues.eventType == null) {
```

Comment:
Instead of displaying an alert. Take advantage of the formErrors object we're building and display an error message below the input on the eventForm.

I know we do it all over the place, but using an alert is an unpleasant user experience (can't style them, can only show one at a time, blocks other actions on the page, etc.). We should try to find better ways to show errors in our app moving forward.